### ISSUE TITLE
Module dependecy

### ISSUE THREAD

**Issue by lipkau** [RamblingCookieMonster/BuildHelpers#46]
The module `Configuration` in version **v0.8** is currently loaded from `./Private/*`.
This is causing some trouble for me: **v0.8 does not support PowerShell Core on non-Windows OS**.

I was unable to reproduce the problem described in the code:
https://github.com/RamblingCookieMonster/BuildHelpers/blob/31e36e292fb28ae3d4c4f51f7aa3498cca4cb2f5/BuildHelpers/BuildHelpers.psm1#L19-L22

Has this been fixed? Is it possible to switch to using `RequiredModules`?

**Comment by RamblingCookieMonster** [comment]
Hiyo!

Thanks for the heads up!  I'd be up for doing this at the next major build, assuming:

* RequiredModules supports nested requirements (e.g. Module X requires Module Y, Module Y requires Module Z - Install-Module resolves X+Y+Z).  This might already be the case, but I vaguely recall it being an issue (and even then, I'd be wary if it was a recent fix)
* Whatever behavior I ran into has been fixed (I would assume this is the case by now).

Until then, it seems the easiest workaround would be to drop in the latest version.  Have a few things I'm working on, might be a little while until I can work on it - feel free to take a stab if this makes sense for you in the interim!

Cheers!
